qendpoint-store/src/main/java/com/the_qa_company/qendpoint/store/EndpointStore.java

```diff
@@ -184,7 +184,7 @@ public EndpointStore(EndpointFiles files, HDTOptions spec, boolean inMemDeletes,
 			throws IOException {
 		// load HDT file
 		this.spec = (spec = HDTOptions.ofNullable(spec));
-		deleteDisabled = spec.getBoolean(OPTION_QENDPOINT_DELETE_DISABLE, false);
+		deleteDisabled = spec.getBoolean(OPTION_QENDPOINT_DELETE_DISABLE, true);
```

Comment:
I must be doing something wrong somewhere. With the HDT and index files I got from Dennis I kept getting an exception related to the part of the code that handles deleted statements. And when I tried to change the config in the application.properties file it didn't seem to make a difference. So I explicitly disabled it in order to be able to test my queries.

Reply:
If you want to set a config, you can add it inside the repo_model.ttl file in the triple
```turtle
mdlc:main mdlc:hdtSpec "qendpoint.delete.disable=true;" .
```

By default, we need the deletes, so the default value should be false.